extensions/agent/src/dialogs/createStepDialog.ts

```diff
@@ -15,6 +15,7 @@ export class CreateStepDialog {
 	// Top level
 	//
 	private static readonly DialogTitle: string = 'New Job Step';
+	private static readonly FileBrowserDialogTitle: string = 'Locate Database Files - ';
```

Comment:
are this and the other strings here going to be localized?

Reply:
Eventually every string in the extension has to be localized. Since we can't import nls and have no example of a localized string in the extension, I'm not sure how to localize them for now.